### DISCUSSION TITLE
Throttle Position Hold for self centering sticks

### DISCUSSION THREAD

**Discussion by relaxibus** [iNavFlight/inav#9140] [Ideas]
I´m building a new rugged RC controller for fixed wing surveying planes. The issue is that rugged joysticks (OTTO, Ruffy, APEM) are all center spring versions. My idea is to add a new Mode "Throttle Hold" activable with a switch. In this way you keep the stick in your desired position and press the switch, now you can release the stick and inav keeps the throttle position. pressing the switch again disengage the mode and you get back full stick authority. This is also very important for safety when you land so you can 0 the throttle stick, press the switch and if you release the center spring stick, the motor stays off.
Is this a feature that makes sense?

**Comment by Jetrell** [comment]
If you only want to fly your surveying in ANGLE or ACRO  modes. Then I suppose that will be fine.

But if you use a navigation mode - CRUISE, COURSE HOLD, WP or LOITER modes. Then there are other factors that influence motor throttle control.  
```
nav_fw_allow_manual_thr_increase 
nav_overrides_motor_stop

nav_fw_pitch2thr
nav_fw_pitch2thr_smoothing
nav_fw_pitch2thr_threshold
nav_fw_climb_angle 
nav_fw_dive_angle
```
They either allow for throttle override in navigation modes. Or control the motors output throttle, based on the planes angle of attack. to prevent a stall when climbing, or to reduce the descent speed.

**Comment by relaxibus** [comment]
Yes. But at the end, what about the support of center spring joysticks for throttle on fixed wings? Something that could be taken into consideration?

**Reply by Jetrell** [reply]
I don't think there is much call for such a feature by the majority of users. Because is could lead to a safety issue. if the pilot had an emergency and momentarily forgot which switch they need to use to disengage the throttle.

But INAV has a programming tab.. And if you have a good understanding of how INAV controls and scales the throttle in all of the flight modes.. Then you could write some logic to accomplish what you require.

**Comment by relaxibus** [comment]
I tried to use the programming tab, but was unable to get to a result. May be some one could help? That would be very appreciated :-)
